systems/controllers/sampling_based_c3_controller.cc

```diff
@@ -407,6 +407,70 @@ SamplingC3Controller::SamplingC3Controller(
     std::cout << "Initial filtered_solve_time_: " << filtered_solve_time_
               << std::endl;
   }
+
+  // Below code loads in the mesh and enumerates triangular faces.
```

Comment:
Is it worth now writing this in a way that only attempts to do this if the sampling strategy requires it?  As written, the jack and push_t examples will no longer run because they will throw this runtime error.  I think it'd be as simple as `if (sampling_params_.sampling_strategy == SamplingStrategy:kMeshNormal)`, then I'd ensure the jack and push_t examples still run.